db/blip_handler.go

```diff
@@ -299,16 +308,19 @@ func (bh *blipHandler) sendChanges(sender *blip.Sender, opts *sendChangesOptions
 
 					var changeRow []interface{}
 					if bh.db.DatabaseContext.Options.UnsupportedOptions.ForceBLIPV3 {
-						deleted := 0
+						deletedFlags := changesDeletedFlag(0)
 						if change.Deleted {
-							deleted += 1
+							deletedFlags += changesDeletedFlagDeleted
```

Comment:
I feel like using the bitwise or operator to set flags (`|=`) is more defensive - it avoids a regression in future where the flag is already set, so `+=` doesn't work as expected.

Reply:
Agreed. Forgot to do this earlier.